tests/unit/components/panel/AttributeSection.spec.js

```diff
@@ -0,0 +1,303 @@
+import { installQuasarPlugin } from '@quasar/quasar-app-extension-testing-unit-jest';
+import { shallowMount } from '@vue/test-utils';
+import AttributeSection from 'src/components/panel/AttributeSection.vue';
+import {
+  ComponentAttribute,
+} from 'leto-modelizer-plugin-core';
+
+installQuasarPlugin();
+
+describe('test component: AttributeSection', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallowMount(AttributeSection, {
+      props: {
+        plugin: {},
+        attribute: {
+          name: 'attribute',
+          type: 'String',
+          value: [],
+        },
+        isRoot: true,
+      },
+    });
+  });
+
+  describe('Test attribute: isObject', () => {
+    it('should be equals to true when attribute is an object', () => {
+      expect(wrapper.vm.isObject).toEqual(false);
+    });
+
+    it('should be equals to false when attribute is not an object', () => {
+      wrapper = shallowMount(AttributeSection, {
+        props: {
+          plugin: {},
+          attribute: {
+            name: 'attribute',
+            type: 'Object',
+            value: '',
+          },
+          isRoot: true,
+        },
+      });
+      expect(wrapper.vm.isObject).toEqual(true);
+    });
+  });
+
+  describe('Test watcher: props.attribute', () => {
```

Comment:
for each tests, can you check the value before changing props?

Reply:
done